### DISCUSSION TITLE
Page not secure and page not redirecting properly

### DISCUSSION THREAD

**Discussion by elisawinograd** [community/community#23149] [Other Feature Feedback, Questions, & Ideas]
<p>Hi I just made a github page which is being redirected to <a href="http://adam-goldstein.com" rel="noopener nofollow ugc">adam-goldstein.com</a>. How can I make the page secure? I also keep getting an error right when I make some changes, something to do with the cookies? I would like to fix that as well, see screenshot attached. Any suggestions appreciated! Thanks so much!</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/adam-goldstein/adam-goldstein.github.io" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:420/420;"><img src="https://avatars.githubusercontent.com/u/22730235?s=400&amp;amp;v=4" class="thumbnail" width="420" height="420"></div>

<h3><a href="https://github.com/adam-goldstein/adam-goldstein.github.io" target="_blank" rel="noopener nofollow ugc">adam-goldstein/adam-goldstein.github.io</a></h3>

  <p><span class="label1">Contribute to adam-goldstein/adam-goldstein.github.io development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/9/296332fcd98b1f5703aedb5d7ddbd30c9c102f7d.png" data-download-href="/uploads/short-url/5U86N2oOKQMlgZ64ubBKp6od73f.png?dl=1" title="Screen Shot 2021-02-13 at 6.20.11 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11314267/181075340-fcf2df55-daa1-44a4-82e1-5daf76a1d40f.png" alt="Screen Shot 2021-02-13 at 6.20.11 PM" data-base62-sha1="5U86N2oOKQMlgZ64ubBKp6od73f" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/296332fcd98b1f5703aedb5d7ddbd30c9c102f7d_2_690x431.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/296332fcd98b1f5703aedb5d7ddbd30c9c102f7d_2_1035x646.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/296332fcd98b1f5703aedb5d7ddbd30c9c102f7d_2_1380x862.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/296332fcd98b1f5703aedb5d7ddbd30c9c102f7d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-13 at 6.20.11 PM</span><span class="informations">2510×1570 242 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mpboom** [comment]
<p>Hey <a class="mention" href="/elisawinograd">@elisawinograd</a> !</p>
<p>About the HTTPS: you still <a href="https://dns.google.com/query?name=adam-goldstein.com" rel="noopener nofollow ugc">seem to have</a> an A record in your DNS which does not point to GitHub pages (but rather <code>162.255.119.20</code>). I am unsure if this is THE problem, but it shouldn’t be there. Try updating that and waiting for a few hours to see if a certificate gets activated on your domain.</p>
<p>About the too-many-redirects issues: it’s important for us to know what kind of changes led to the situation. Could you for example point us to a commit that caused the trouble?</p>

**Comment by elisawinograd** [comment]
<p>Thanks for your help. How can I get rid of this A record? I cannot find it where I am hosting my domain (see attached).</p>
<p>Basically all recent commits lead to the too-many-redirects issue. Any clues?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/c/dcf0fc55ae093b842e28725b44d238e762f5b960.png" data-download-href="/uploads/short-url/vwxlSHYIeyzMC0BfZjDoA5BHdZe.png?dl=1" title="Screen Shot 2021-02-14 at 6.58.38 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/11314267/181075344-aa727ea0-7ef8-4f7a-a2dc-1625fc71316d.png" alt="Screen Shot 2021-02-14 at 6.58.38 PM" data-base62-sha1="vwxlSHYIeyzMC0BfZjDoA5BHdZe" width="690" height="321" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcf0fc55ae093b842e28725b44d238e762f5b960_2_690x321.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcf0fc55ae093b842e28725b44d238e762f5b960_2_1035x481.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcf0fc55ae093b842e28725b44d238e762f5b960_2_1380x642.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/c/dcf0fc55ae093b842e28725b44d238e762f5b960_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-14 at 6.58.38 PM</span><span class="informations">1886×880 63 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment]
<p>It’s possibly that URL Redirect record that’s adding the extra <code>A</code> record.</p>
<p>That’s also probably giving you the “too many redirects” problem as it’s redirecting visitors to the www variant of your domain and then GitHub is automatically redirecting them back to the non-www version.</p>

**Comment by mpboom** [comment]
<p>Yes, I do think <a class="mention" href="/thomasshaped">@thomasshaped</a> is right. The “URL redirect record” is probably the culprit in both the HTTPS issues as well as the redirect loops.</p>

**Comment by elisawinograd** [comment]
<p><a class="mention" href="/mpboom">@mpboom</a> <a class="mention" href="/thomasshaped">@thomasshaped</a> thank you so much! I actually think that solved the redirecting issue. I will check in next time a do a big commit. The security issue is still there but I just enforced HTTPS so I hope this fixes it :slight_smile:</p>

**Comment by otter-computer** [comment]
<p>If you managed to check the “Enforce HTTPS” checkbox then I believe that means a valid cert has been generated for your site too :raised_hands: :tada: Fingers crossed it has worked!</p>

**Comment by elisawinograd** [comment, marked as answer]
<p><a class="mention" href="/thomasshaped">@thomasshaped</a> <a class="mention" href="/mpboom">@mpboom</a> security issue seems to be fine now! thanks so much for your help :innocent:</p>
